src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -660,11 +660,15 @@ private void removeTerritoriesWhereTransportsAreExposed() {
             ProBattleUtils.calculateBattleResults(player, t, patd.getUnits(), patd.getMaxEnemyDefenders(player, data),
                 patd.getBombardTerritoryMap().keySet(), true);
         int production = 0;
+        int isEnemyCapital = 0;
         final TerritoryAttachment ta = TerritoryAttachment.get(t);
         if (ta != null) {
           production = ta.getProduction();
+          if (ta.isCapital()) {
+            isEnemyCapital = 1;
+          }
         }
-        final double attackValue = result.getTUVSwing() + production;
+        final double attackValue = result.getTUVSwing() + production * (1 + 3 * isEnemyCapital);
```

Comment:
I'll move it into the isEnemyCapital variable but to some extent these are arbitrary weights right now. Mostly just a stop gap so the AI doesn't bail on amphib attacking capitals.